API-GUIDELINE.md

```diff
@@ -0,0 +1,254 @@
+#API GUIDELINES
```

Comment:
```suggestion
# API GUIDELINES
```
I think you need a space after all `#`, `##`...

Reply:
I saw that too 🤔 but after testing it worked on markdown intellij

Reply:
I can confirm: in the Preview nothing change, every paragraph doesn't have the '#'

Reply:
thx for checking.
I'would still prefer with a whitespace. It's easier to read the markdown.

Reply:
ok I'm gonna update for each of them